xml/schema/ACL/Cache.xml

```diff
@@ -55,7 +55,7 @@
   <field>
     <name>modified_date</name>
     <title>Cache Modified Date</title>
-    <type>date</type>
+    <type>timestamp</type>
```

Comment:
I'm still reading, and this new code seems more intuitive/sensible... that old schema had it using `date`, which implies that TTL was a multiple of 24 hours, which feels surprising/weird/design-mistakey (unless there's something special I'm not aware of).